src/decorator/options/ColumnCommonOptions.ts

```diff
@@ -47,4 +49,10 @@ export interface ColumnCommonOptions {
      */
     isArray?: boolean;
 
-}
\ No newline at end of file
+    /**
+     * Specifies a value transformer that is to be used to (un)marshal
+     * this column when reading or writing to the database.
+     */
+    valueTransformer?: ValueTransformer<any, any>;
```

Comment:
What do you think if we just name it `transform`?

Reply:
@pleerock Sounds good to me. `valueTransformer` is indeed a bit too long.